src/main/java/com/example/flight/controllers/AirportController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.flight.controllers;
+
+import java.util.List;
+
+import com.example.flight.entities.Airport;
+import com.example.flight.exceptions.AirportNotFoundException;
+import com.example.flight.services.AirportService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/airports")
+class AirportController {
+
+    private final AirportService service;
+
+    @Autowired
+    private AirportController(AirportService service) {
+        this.service = service;
+    }
+
+    @GetMapping
+    public List<Airport> getAllAirports() {
+        return service.getAllAirports();
+    }
+
+    @PostMapping
+    public Airport postAirport(@RequestBody Airport airport) {
```

Comment:
I know that I suggested `actionMethodName` but I prefer readability for this case.

Which is `createAirport`, however, my question remains. Why would a user create an airport??